### PR TITLE
Sinai 4 BAVet15

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Sinai 4, a complicated manuscript, 
not everything can be clarified for the moment.

### CODE REVIEW

File: SinaiStCatherine/SinaiEt004.xml [hunk 1]
```diff
@@ -0,0 +1,894 @@
...
+                                    <item xml:id="q14" n="13">
+                                        <dim unit="leaf">12</dim>
+                                        <locus from="103r" to="114v"/>
+                                      <note>Structure uncertain</note>
+                                    </item>
+                                    
+                                    <item xml:id="q15" n="14">
+                                        <dim unit="leaf">11</dim>
+                                        <locus from="115r" to="125v"/>
+                                      <note>Structure uncertain</note>
```
Comment (eu-genia) [inline SinaiStCatherine/SinaiEt004.xml:128]: if no info on s.l. is given the script for "drawing" quires does not work.

File: SinaiStCatherine/SinaiEt004.xml [hunk 2]
```diff
@@ -0,0 +1,894 @@
...
+                                    </item>
+                                    
+                                    <item xml:id="q16" n="15">
+                                        <dim unit="leaf">9</dim>
+                                        <locus from="126r" to="134v"/>
+                                      <note>Structure uncertain</note>
```
Comment (eu-genia) [inline SinaiStCatherine/SinaiEt004.xml:134]: same

File: SinaiStCatherine/SinaiEt004.xml [hunk 3]
```diff
@@ -0,0 +1,894 @@
...
+                        <measure unit="quire">2</measure>
+                        <dimensions type="leaf" unit="mm">
+                          <height cert="low">120</height>
+                          <width cert="low">80</width>                            
+                        </dimensions>
+                      </extent>
+                      <collation>
+                        <note><ref target="#p3">Unit 3</ref> is composed of 
+                         <ref target="#q17">quire 17</ref>
+                          as <ref target="#q18">quire 18</ref>.</note>
```
Comment (eu-genia) [inline SinaiStCatherine/SinaiEt004.xml:690]: as -> and?

File: SinaiStCatherine/SinaiEt004.xml [hunk 4]
```diff
@@ -0,0 +1,894 @@
...
+                </physDesc>                        
+                
+                <history>
+                  <origin>
+                    <origDate notBefore="1350" notAfter="1450">Some time around or possibly 
+                      before 1428</origDate>. 
+                    <ref target="#p3">Unit 3</ref> is possible a fragment of 
+                    a manuscript. A commissioner or owner, Ṗeṭros (who could be <persName ref="PRS13951Petros">the priest Ṗeṭros</persName> 
+                    brother of <persName ref="PRS13957Qalementos"/>, see above) 
+                    is mentioned in the supplication formulas (for instance <locus target="#135va #136ra #137ra "/>).
```
Comment (eu-genia) [inline SinaiStCatherine/SinaiEt004.xml:749]: remove blank space after 137ra

**Review by eu-genia — APPROVED** [review]
ok just small things maybe to consider

File: SinaiStCatherine/SinaiEt004.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:128]: Yes, but I cannot decipher the structure of this part on photos.

File: SinaiStCatherine/SinaiEt004.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:134]: I will try to clarify it later.

File: SinaiStCatherine/SinaiEt004.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:690]: Corrected.

File: SinaiStCatherine/SinaiEt004.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:749]: Done.

File: SinaiStCatherine/SinaiEt004.xml [hunk 5]
```diff
@@ -0,0 +1,894 @@
...
+                              <msItem xml:id="p1_i1.3.15">
+                        <locus target="#124v"/>
+                        <title type="complete">Stichometry of the Psalms</title>     
+                    </msItem>
+                    </msItem>
+
+                            <msItem xml:id="p1_i1.4">
+                      <locus from="124va" to="134vb"/>
+                              <title type="complete" ref="LIT1961Mashaf" cert="medium">A collection of readings for canonical hours, in part similar to that 
+                              in <ref type="mss" corresp="VATet15"/></title>
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt004.xml:319]: BAVet15

File: SinaiStCatherine/SinaiEt004.xml [hunk 6]
```diff
@@ -0,0 +1,894 @@
...
+                                      ይጸርኁ፡ ወይብሉ፡ ካልእ፡ ለካልዑ፡ ኪሩቤል፡ ይብሉ፡ ቅዱስ፡ ወሱራፌል፡ ይብሉ፡ እኩት፡ ወሩፋኤል፡ ይብሉ፡ ቡሩክ፡
+                                    </q>  
+                                  </item>
+                                  
+                                  
+                                  <item xml:id="a5">
+                                    <locus target="#179r"/>
+                                    <desc type="CommemorativeNote">A note recording the
+                                      death of <persName ref="PRS13951Petros">the priest Ṗeṭros</persName> 
+                                      (obviously the same person as mentioned in <ref type="mss" corresp="SinaiEt004"/>) that
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt004.xml:506]: If it is obvious, probably "obviously" is not needed in the sentence

File: SinaiStCatherine/SinaiEt004.xml [hunk 7]
```diff
@@ -0,0 +1,894 @@
...
+
+                      </list>
+                  </additions>                  
+               </physDesc>                        
+                        
+               <history>
+                 <origin>
+                   <origDate notBefore="1400" notAfter="1450">Some time after 1428</origDate>.
+                   <ref target="#p2"/> was written some time after <ref target="#p1"/>, most probably outside Ethiopia
+                   (as the writing support is paper).  <ref target="#2"/> was unlikely meant as a circulation unit.
```
Comment (thea-m) [inline SinaiStCatherine/SinaiEt004.xml:642]: p2?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: SinaiStCatherine/SinaiEt004.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:319]: OK

File: SinaiStCatherine/SinaiEt004.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml:506]: ok

File: SinaiStCatherine/SinaiEt004.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiEt004.xml]: yes.

**Pull request merged**